### PR TITLE
Remove references to the milestone burndown Google Group

### PR DESCRIPTION
**Pull request by justaugustus**
Closes: #440 

Signed-off-by: Stephen Augustus <saugustus@vmware.com>
/hold

### CODE REVIEW

**Comment by justaugustus** [comment]
Announcement to move forward with this on k-dev: https://groups.google.com/d/topic/kubernetes-dev/QIfxAp0xv98/discussion

**Comment by justaugustus** [comment]
Tagging for visibility:
@claurence @lachie83 @nikopen @kacole2 

For approval:
/assign @tpepper @jberkus @spiffxp

File: releases/EXCEPTIONS.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ To file for an exception, please fill out the questions below:
 
 Email them to:
 
-- Your SIG lead
-- The Release Team Lead
-- kubernetes-milestone-burndown@googlegroups.com
+- Your SIG's mailing list
+- kubernetes-release-team@googlegroups.com
+- kubernetes-sig-release@googlegroups.com
```
Comment (tpepper) [inline releases/EXCEPTIONS.md]: Could you also update the example template below for its yaml "thread: " lines?

**Pull request merged**